javascript/ql/src/experimental/Security/CWE-347-noVerification/jwtNoVerification.qhelp

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE qhelp PUBLIC "-//Semmle//qhelp//EN" "qhelp.dtd">
```

Comment:
`.qhelp` files are supposed to have the same basename as a `.ql`.  
So for this `.qhelp` file we would expect to find a corresponding `jwtNoVerification.ql` file. 

You need one `.qhelp` file pr. `.ql` fille, but you can use `.inc.qhelp` files to make reusable snippets.